src/scitokens/utils/config.py

```diff
@@ -29,13 +29,13 @@ def set_config(config = None):
     global configuration # pylint: disable=C0103
 
     if isinstance(config, six.string_types):
-        configuration = configparser.SafeConfigParser(CONFIG_DEFAULTS)
+        configuration = configparser.SafeConfigParser(CONFIG_DEFAULTS, allow_no_value=True)
```

Comment:
This breaks Python 2.6 compatibility ... which was already broken, right?  If so, that's OK.